compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -57,6 +57,11 @@ llvm::cl::opt<bool> clGPUTestTileAndFuseVectorize(
         "test the tile and fuse pipeline for all supported operations"),
     llvm::cl::init(false));
 
+llvm::cl::opt<bool> clUseLLVMGPUVectorizePipeline(
```

Comment:
The variable name and flag name are different. That's confusing.

Reply:
Can we make the flag variable name and the flag itself something more manageable/consistent. "iree-codegen-llvmgpu-use-legacy-default-pipeline" is a bit of a mouthful.